Forms automated test - Master | LPS-155760

vsigor opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

rafaellajordao commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-26.liferay.com/job/test-portal-source-format/1408/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/1408/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9343a25f6b971908978fa61c87bdd11468aa9d22">9343a25f6b971908978fa61c87bdd11468aa9d22</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/rafaellajordao/liferay-portal/tree/LRQA-65623-3">LRQA-65623-3</a><br/>Branch GIT ID: <a href="https://github.com/rafaellajordao/liferay-portal/commit/2273fb90ef306e47d9a3918ce01b4c165d004cd9">2273fb90ef306e47d9a3918ce01b4c165d004cd9</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-source-format/1408/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/1408/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-source-format/1408/">test-portal-source-format#1408</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/1408/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/rafaellajordao/liferay-portal/pull/7">rafaellajordao#7</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1609505977">[master] ci:test:sf - rafaellajordao &gt; rafaellajordao - PR#7 - 2022-02-09[11:27:03]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/4936/">publish-testray-report#4936</a><br/></div></div>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-source-format/1408/">test-portal-source-format#1408</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/1408/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/rafaellajordao/liferay-portal/pull/7">rafaellajordao#7</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1609505977">[master] ci:test:sf - rafaellajordao &gt; rafaellajordao - PR#7 - 2022-02-09[11:27:03]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/4937/">publish-testray-report#4937</a><br/></div></div>

rafaellajordao commented:
ci:test:forms-functional:nocompile

liferay-continuous-integration commented:
The test will run with a prebuilt bundle.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>forms-functional</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/1645/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1645/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:forms-functional - 2 out of 2 jobs passed in 39 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9343a25f6b971908978fa61c87bdd11468aa9d22">9343a25f6b971908978fa61c87bdd11468aa9d22</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9343a25f6b971908978fa61c87bdd11468aa9d22">9343a25f6b971908978fa61c87bdd11468aa9d22</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//2906">Acceptance Upstream DXP (master) #2906</a></p><details><summary><strong>ci:test:forms-functional - 2 out of 2 jobs PASSED</strong></summary><details><summary><strong>2 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/1645/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/475633/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1645/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/1645/">test-portal-acceptance-pullrequest(master)#1645</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1645/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>forms-functional<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/rafaellajordao/liferay-portal/pull/7">rafaellajordao#7</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1609530202">[master] ci:test:forms-functional - rafaellajordao &gt; rafaellajordao - PR#7 - 2022-02-09[12:11:06]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/4886/">publish-testray-report#4886</a><br/></div></div>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/1645/">test-portal-acceptance-pullrequest(master)#1645</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1645/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>forms-functional<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/rafaellajordao/liferay-portal/pull/7">rafaellajordao#7</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1609530202">[master] ci:test:forms-functional - rafaellajordao &gt; rafaellajordao - PR#7 - 2022-02-09[12:11:06]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/4887/">publish-testray-report#4887</a><br/></div></div>

ces-sontruong committed:
LPS-106788 Escape selected site name

cgoncas committed:
LPS-152539 Add getConfigurationURL method to SegmentsConnfigurationProvider:

This method will return the URL where the user can configure segments configuration.

cgoncas committed:
LPS-152539 Implement the new method.

Wwe will redirect the user to the Instance Settings configuration.

cgoncas committed:
LPS-152539 Create a new method in the Display COntext of roles that return the configuretion URL.

cgoncas committed:
LPS-152539 Add a link to the configuration URL in the warning message.

cgoncas committed:
LPS-152539 Ad tests for getConfigurationURL

SelenaAungst committed:
LPS-153090 Changed style=warning to displayType=warning

CarlosBrichete committed:
LPS-152539 Modify poshi test according to the changes

ling-alan-huang committed:
LPS-147681 Add javax.portlet.version=3.0

CrescenzoRegaLF committed:
COMMERCE-8676 Add duplicate shipment item exception

CrescenzoRegaLF committed:
COMMERCE-8676 Add check to prevent ERC field duplication error

CrescenzoRegaLF committed:
COMMERCE-8676 Test method added

CrescenzoRegaLF committed:
COMMERCE-8676 Autogenerated code (gw buildService)

CrescenzoRegaLF committed:
COMMERCE-8676 Added new localized key

CrescenzoRegaLF committed:
COMMERCE-8676 Added edit commerce shipment item external reference code action command

CrescenzoRegaLF committed:
COMMERCE-8676 Added shipment item external reference code to provider and fds view

CrescenzoRegaLF committed:
COMMERCE-8676 Autogenerated code (gw buildLang)

baharturk committed:
COMMERCE-8676 Fix functional tests

Commit:
LPS-155447 Create Unit Test for Search bar on Home Page

Commit:
LPS-155447 Create Unit Test for Search Header component on Home Page

robertoDiaz committed:
LPS-155882 Get the available readers and use them (logic based in javax.imageio.ImageIO.doWrite)

Commit:
LPS-155447 Fix tests on Search Header component

Commit:
LPS-155447 Create tests for Project Card component on Home Page

adolfopa committed:
LPS-154164 Support auto tagging on updates for journal

Commit:
LPS-155447 Create unit tests for Project List component for Home Page

baharturk committed:
COMMERCE-8676 Add assertOptionsInShipmentSteps macro

baharturk committed:
COMMERCE-8676 Add paths for shipment item erc

baharturk committed:
COMMERCE-8676 Add CanEditShipmentItemERC and CanNotUseTheSameERCForDifferentShipmentItems tests

baharturk committed:
COMMERCE-8676 Auto SF

baharturk committed:
COMMERCE-8676 Make changes after review

balazssk committed:
LPS-156202 Test - that checkbox typed ExpandoColumn keeps its defaultData if imported after it was removed

balazssk committed:
LPS-156202 Skip validating column type to get defaultValue during import, as validation requires the ExpandoColumn to already exist

IstvanD committed:
LPS-155916 Cannot set Field Group with a nested Date Field repeatable

thaynam committed:
LPS-155581 Delete default files

thaynam committed:
LPS-155581 Create Style Books

thaynam committed:
LPS-155581 Create Guest Page

thaynam committed:
LPS-155581 Create Guest Template

thaynam committed:
LPS-155581 Add Guest images

thaynam committed:
LPS-155581 Create Guest fragments

thaynam committed:
LPS-155581 Fix texts in guest page

thaynam committed:
LPS-155581 Set style-book as default

thaynam committed:
LPS-155581 Fix font path

adolfopa committed:
LPS-150285 Restrict criterion to images

jakesliwka committed:
LPS-155839: Templates no longer displaying in escaped format, not necessary for how they are being used

Commit:
COMMERCE-9513 Fix AssertProductsCannotBeImportedIfASubscriptionProductIsInTheOrder Test

Commit:
COMMERCE-9513 Add VIEW_ITEM_FROM_LIST path

Commit:
COMMERCE-9513 Fix CommerceDeliveryDate setUp

ealonso committed:
LPS-156092 Don't render the form container if the info form is null

Commit:
LPS-155447 Fix constants aproach to test Project List component

Commit:
LPS-155447 Fix constants aproach to test Project card component

Commit:
LPS-155447 Fix constants aproach to test SearchHeader component

Commit:
LPS-155447 Fix constants aproach to test Searchbar component

Commit:
COMMERCE-9513 Fix DiscontinuedProductIsReplacedWhenImportedFromOrders Test

Commit:
COMMERCE-9513 Fix DiscontinuedProductIsReplacedWhenImportedFromOrders Test

Commit:
COMMERCE-9513 Fix ReplacementProductInformationIsDisplayedInProductDetails Test

Commit:
LPS-156264 Consider CHECKBOX_MULTIPLE DDM form field type to create the correspondent entity field to allow filtering through the API. In this case, a StringEntityField

CrescenzoRegaLF committed:
COMMERCE-8676 Refactor of ERC field duplication control

CrescenzoRegaLF committed:
COMMERCE-8676 semantic versioning (gw baseline)

CrescenzoRegaLF committed:
COMMERCE-8376 Fix functional tests

veroglez committed:
LPS-156297 Add icon for required field with the corresponding marked

sandrodw3 committed:
LPS-156092 Move to FormService

sandrodw3 committed:
LPS-156092 Rename to formTypes

sandrodw3 committed:
LPS-156092 Use constant

sandrodw3 committed:
LPS-156092 Make sure type and subtype exist when checking whether a form is mapped or not

sandrodw3 committed:
LPS-156092 Ensure form types are loaded if FormWithControls is rendered

sandrodw3 committed:
LPS-156092 Check type and subtype exist, otherwise show None option

sandrodw3 committed:
LPS-156092 Remove extra component

Kvale1733 committed:
COMMERCE-9530 Add openCatalogsAdmin macro

Kvale1733 committed:
COMMERCE-9530 Add UserWithoutPermissionCannotAddEditOrDeleteCatalogs test

adolfopa committed:
LPS-150285 Baseline

Kvale1733 committed:
COMMERCE-9531 Add MasterCatalogIsCreatedByDefaultAndCannotBeDeleted test

Commit:
LCD-14360 avoid NPE when config map exists

Kvale1733 committed:
COMMERCE-9532 AddCatalogProductsPromotionsAndPriceListsAreLocalized test

sandrodw3 committed:
LPS-156092 Initialize cache in index.js instead of App.js

Hijky committed:
LPS-156371 Add a test to can get ManyToMany relationship details as nested fields_3

jkappler committed:
LPS-156486 Method already checks for this, so we can remove it

ealonso committed:
LPS-146828 SF - Inline

ealonso committed:
LPS-146828 SF

ealonso committed:
LPS-146828 SF - Unnecessary exception

ealonso committed:
LPS-146828 SF - Adds diamond operator

Commit:
LPS-156483 Refactor cases to adapt new behavior

ling-alan-huang committed:
LCD-14368 Move *.war to dependencies folders

victorg1991 committed:
LPS-146828 Not needed

victorg1991 committed:
LPS-146828 Rename

ealonso committed:
LPS-155839 Avoid doble escape since we already are escaping the title on vertical card taglib, see LPS-124163

victorg1991 committed:
LPS-156489 Set the baseURL to blank instead of new line

victorg1991 committed:
LPS-156489 Check correct attribute

izaera committed:
LPS-156488 Make CETFactory depend on all CETImplFactories

Currently CETFactory is tracking available CETImplFactories using a
ServiceTrackerMap. This was good to make implementing a new CET easier
because the implementer didn't need to provide any extra machinery to wire
things up.

However, because CETFactory does not depend directly on CETImplFactories,
it may be started way before the CETImplFactories which leads to exceptions
when trying to instantiate CETs.

Also, because the LocalServiceImpl is sometimes started before the
CETImplFactories and it tries to deploy all remote apps, this wiring leads
to failure due to above mentioned exceptions.

The solution to this is to make CETFactory depend on every CETImplFactory
so that the latter are started before the former.

I'm creating a separate Component for this to ease implementation of new
CETs and keep the responsibilities isolated. I think that, if we continue
putting common code in CETFactoryImpl we will grow that class huge and
will be hardly maintainable and very prone to Git conflicts. I'm using
reflection in CETImplFactoryRegistry too, for the very same reasons.

izaera committed:
LPS-147618 Escape URL fields to avoid XSS attacks

This implementation is very safe, because it makes any data read from DB be
escaped. The "downside" is that the real data the user typed in the UI
won't ever be accessible, but I don't think we need that for anything...

ozysouza committed:
LPS-156183 Switch pause for WaitForElementPresent in testcase

ozysouza committed:
LPS-156183 Add comment to explain pause

victorg1991 committed:
LPS-146828 Rename prop since it is only used here

Commit:
LPS-156264 Ignore flaky integration test. I've investigated for a long time and seems to be related with the core infrastructure as the default user value is not refreshed immediately in some cases when the default languages are changed

ling-alan-huang committed:
LCD-14368 Exclude */modules/apps/static/*

thaynam committed:
LPS-155581 SF

sandrodw3 committed:
LPS-156092 Adapt tests

Commit:
LPS-155447 Fix tests props on Project List and Project Card components

brianchandotcom committed:
LCD-14360 SF

brianchandotcom committed:
LPS-155882 SF/Wordsmith

brianchandotcom committed:
LPS-152539 #sfme

brianchandotcom committed:
LPS-152539 Simplify / as used

gamerson committed:
LPS-155410 make sure to undeploy any previous CET registrations

brianchandotcom committed:
LPS-156488 Simplify. If I have to add a @Reference for every factory impl, we might as well just do it like this and not pay for the OSGi overhead

brianchandotcom committed:
LPS-155447 upload

brianchandotcom committed:
COMMERCE-8676 Wordsmith

brianchandotcom committed:
Revert "LPS-156264 Ignore flaky integration test. I've investigated for a long time and seems to be related with the core infrastructure as the default user value is not refreshed immediately in some cases when the default languages are changed"

This reverts commit 78669227db5940c5b1f5e3791eb26061852d173c.

Commit:
LPS-155760 Add ViewSelectFromListFieldDetails test
